.github/workflows/markdown-style-checks.yml

```
@@ -0,0 +1,24 @@
+name: Markdown style checks
+
+on:
+  push:
+    branches:
+    - main
+    paths:
+    - '/**'   # Removed docs as everything is in the root directory
+  pull_request:
+    branches:
+    - '*'
+    paths:
+    - '/**'   # Removed docs as everything is in the root directory
+
+jobs:
+  markdown-lint:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: DavidAnson/markdownlint-cli2-action@v16
```

Is this action approved ? Asking because of the recent incident and @morphis mentioning in the standup that only canonical developed or standard gh actions should be used from now one ?

---

Since it is coming from the upstream starter pack, I checked it out. Looks like it is available on the GH marketplace, actively maintained. But I have asked the SP maintainer about this too.

---

Good point - I think this should be pinned to a hash.

---

v16 is commit [b4c9feab76d8025d1e83c653fa3990936df0e6c8](https://github.com/DavidAnson/markdownlint-cli2-action/commit/b4c9feab76d8025d1e83c653fa3990936df0e6c8)
```suggestion
    - uses: DavidAnson/markdownlint-cli2-action@b4c9feab76d8025d1e83c653fa3990936df0e6c8 # v16.0.0
```